Packs/Arcanna/Scripts/ArcannaFeedbackPostProcessingScript/ArcannaFeedbackPostProcessingScript.yml

```diff
@@ -0,0 +1,28 @@
+args:
+- default: false
+  description: '[Enter a description of the argument, including any important information
+    users need to know, for example, default values.]'
+  isArray: false
+  name: dummy
+  required: true
+  secret: false
+comment: '[Enter a description of the script, including what function it performs
+  and any important information users need to know, for example required permissions.]'
+commonfields:
+  id: ArcannaFeedbackPostProcessingScript
+  version: -1
+enabled: false
+name: ArcannaFeedbackPostProcessingScript
+outputs:
+- contextPath: BaseScript.Output
+  description: '[Enter a description of the data returned in this output.]'
+  type: String
+script: '-'
+system: false
+tags:
+- basescript
+timeout: '0'
+type: python
+subtype: python3
+dockerimage: demisto/python3:3.9.5.21272
+fromversion: '6.0.0'
```

Comment:
It seems like the yml file is the default one that was generated. Please remove it or change it for your own use.

Reply:
Done